x/evm/types/expected_keepers.go

```diff
@@ -45,9 +45,9 @@ type ChainKeeper interface {
 	GetChainID(ctx sdk.Context) (*big.Int, bool)
 	GetRequiredConfirmationHeight(ctx sdk.Context) (uint64, bool)
 	GetRevoteLockingPeriod(ctx sdk.Context) (int64, bool)
-	GetGatewayByteCodes(ctx sdk.Context) ([]byte, bool)
-	GetBurnerByteCodes(ctx sdk.Context) ([]byte, bool)
-	GetTokenByteCodes(ctx sdk.Context) ([]byte, bool)
+	GetGatewayByteCode(ctx sdk.Context) ([]byte, bool)
+	GetBurnerByteCode(ctx sdk.Context) ([]byte, bool)
+	GetTokenByteCode(ctx sdk.Context) ([]byte, bool)
```

Comment:
why not have one for the absorver too?

Reply:
I did it like https://github.com/axelarnetwork/axelar-core/blob/c10d4737a161665b7344f10f60c705e7337b9b00/x/evm/keeper/chainKeeper.go#L273